docs/how-to/configuring-runtime-options.rst

```diff
@@ -1205,8 +1205,8 @@ Sample text configuration file
 -----------------------------------
 
 Text files support very basic variables and are case-insensitive.
-Variables are created when an lvalue starts with a ``$`` and are
-dereferenced when they appear as rvalues.
+Variables are created when an left value starts with a ``$`` and are
```

Comment:
lvalue and rvalue are the correct terms. Try googling "lvalue" and "left value" and see which returns the correct results